specification/common-types/resource-management/v3/types.json

```diff
@@ -406,29 +539,17 @@
         },
         "etag": {
           "type": "string",
-          "description": "The etag field is *not* required. If it is provided in the response body, it must also be provided as a header per the normal etag convention.  Entity tags are used for comparing two or more entities from the same requested resource. HTTP/1.1 uses entity tags in the etag (section 14.19), If-Match (section 14.24), If-None-Match (section 14.26), and If-Range (section 14.27) header fields. ",
+          "description": "The etag field is *not* required. If it is provided in the response body, it must also be provided as a header per the normal etag convention.\nEntity tags are used for comparing two or more entities from the same requested resource. HTTP/1.1 uses entity tags in the etag (section 14.19),\nIf-Match (section 14.24), If-None-Match (section 14.26), and If-Range (section 14.27) header fields.",
           "readOnly": true
         },
         "identity": {
-          "allOf": [
-            {
-              "$ref": "#/definitions/Identity"
-            }
-          ]
+          "$ref": "#/definitions/Identity"
```

Comment:
It's breaking change too.

Reply:
can you explain a little bit deeper what is the difference between "allOf" and "ref"? my understanding is that in most cases they resolve the same, there are exceptional cases in where autorest doesn't resolve them in the same way but in most cases it does, we tried to reduced this difference to the minimum so if this needs to be re-reviewed I want to have a deeper understanding of the difference

Reply:
Oh yes they are the same. It's reference. I thought it is definition.